cedar-policy-core/src/parser/err.rs

```diff
@@ -68,31 +91,275 @@ impl ParseErrors {
     }
 }
 
-impl std::fmt::Display for ParseErrors {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", MultipleParseErrors(&self.0))
+impl Display for ParseErrors {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match &self.0[..] {
+            [err] => Display::fmt(err, f),
+            _ => write!(f, "multiple parse errors"),
```

Comment:
is this ok? don't we want to display what the errors were, in the multiple-errors case?

Reply:
The `Diagnostic::related` implementation for `ParseErrors` takes care of that: https://github.com/cedar-policy/cedar/pull/148/files/bd323f02f4b36018b5e3e5484a81896825d7bd04#diff-a0c6854c9e052ae63e9897a05331af59064d413ab7f360f54b41e762af89e862R106-R111

The sample image from the PR has an example of how this looks: https://user-images.githubusercontent.com/7958605/250595855-bbd5ac88-b993-4497-a67b-c8823cd72bd0.png

Including the source errors in the `Display` instance would print them out where `multiple parse errors` is, then duplicate them when the source errors are printed out below that. In general, the `Display` trait for an error should cover that error type itself, and let `Error`/`Diagnostic` instances direct error formatters how to dig deeper into root causes.